packages/orchestration/src/util/masking.ts

```diff
@@ -14,9 +14,15 @@ export function buildDpiMaskingProvider(
   return {
     type: 'sap_data_privacy_integration',
     method,
-    entities: entities.map(entity => ({
-      type: entity
-    })),
+    entities: entities.map(entity => {
+      if (typeof entity === 'string') {
+        return { type: entity };
+      }
+      return entity.type === 'custom'
+        ? // eslint-disable-next-line @typescript-eslint/no-unused-vars
+          (({ type, ...rest }) => rest)(entity)
```

Comment:
```suggestion
        ? (({ type: _, ...rest }) => rest)(entity)
```
[pp] I would rename it to `_` and add the following rule to `@typescript-eslint/no-unused-vars` in `eslint.config.js`

` 'argsIgnorePattern': '^_'`

I did add `varsIgnorePattern` before but that was clearly not enough 😄